web/impact/impact/tests/test_functional_expertise_detail_view.py

```diff
@@ -0,0 +1,64 @@
+# MIT License
+# Copyright (c) 2017 MassChallenge, Inc.
+
+import json
+from jsonschema import Draft4Validator
+from django.urls import reverse
+
+from impact.tests.factories import FunctionalExpertiseFactory
+from impact.tests.api_test_case import APITestCase
+from impact.tests.utils import assert_fields
+from impact.v1.views import FunctionalExpertiseDetailView
+
+FUNCTIONAL_EXPERTISE_GET_FIELDS = [
```

Comment:
Not a blocker, but this could be `MPTT_FIELD.keys()`.  However, I can see arguments for keeping them distinct.